src/components/custom-constellation/template/HMRC_ODX_GDSCheckAnswersPage/index.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { useRef, useEffect } from 'react';
+import { getInstructions } from './utils';
+import type { PConnProps } from '@pega/react-sdk-components/lib/types/PConnProps';
+
+import StyledHmrcOdxGdsCheckAnswersPageWrapper from './styles';
+
+interface HmrcOdxGdsCheckAnswersPageProps extends PConnProps {
+  // If any, enter additional props that only exist on this componentName
+  NumCols?: string;
+  instructions?: string;
+  // eslint-disable-next-line react/no-unused-prop-types
+  children: Array<any>;
+}
+
+export default function HmrcOdxGdsCheckAnswersPage(props: HmrcOdxGdsCheckAnswersPageProps) {
+  // template structure setup
+  const { getPConnect, NumCols = '1' } = props;
+  const instructions = getInstructions(getPConnect(), props.instructions);
+
+  let divClass: string;
+
+  const numCols = NumCols || '1';
+  switch (numCols) {
+    case '1':
+      divClass = 'psdk-default-form-one-column';
+      break;
+    case '2':
+      divClass = 'psdk-default-form-two-column';
+      break;
+    case '3':
+      divClass = 'psdk-default-form-three-column';
+      break;
+    default:
+      divClass = 'psdk-default-form-one-column';
+      break;
+  }
+
+  const arChildren = getPConnect().getChildren()[0].getPConnect().getChildren();
+  const dfChildren = arChildren.map((kid, idx) => {
+    kid.key = idx;
+    // @ts-ignore
+    return getPConnect().createComponent(kid.getPConnect().getRawMetadata());
+  });
+
+  // Create a ref to the mainer rendering container
+  const dfChildrenContainerRef = useRef(null);
+
+  // function getSummaryListRows(htmlString) {
```

Comment:
Should this code be removed? Or are we keeping it in case it's needed in the future?

Reply:
Keeping it for now, will do a clean up task of the CYA after today's deployment